in-toto-spec.md

```diff
@@ -989,44 +1067,48 @@ to further verify accompanying metadata.
    each step.
 1. Once all the steps are performed, the final product is shipped to the
    client.
-1. in-oto's verification tools are run on the final product.
-1. in-oto inspects the final product to find a root.layout file that describes
+1. in-toto's verification tools are run on the final product.
+1. If all these steps are successful, verification is done. The user can now
+   use the final product.
+
+### 5.2 Verifying the final product
+  
+ 
+The following algorithm contains an in-depth description of the verification
+procedure.
+ 
+1. Toto inspects the final product to find a root.layout file that describes
    the top-level layout for the project. The signature(s) on the file are
-checked using previously-acquired project owner public key(s). Subsequently, if
-the layout signature verification passes, the functionaries’ public keys are
-loaded from the layout.
+checked using previously-acquired project owner public key(s).
 1. The expiration time is verified to ensure that this layout is still fresh.
+   If the system's date is newer than the expiration date on the layout's
+expiration field, verification should fail.
+1. Subsequently, if the layout signature and expiration are valid, the
+   functionaries’ public keys are loaded from the root.layout in the pubkeys
+field.
 1. The steps, as defined in the layout, are loaded. For each step in the
-   layout, one or more pieces of link or layout metadata is loaded.
-1. If the file loaded metadata is a link metadata file, a data structure
+   layout, one or more pieces of link or layout metadata is loaded. 
+  1. If the file loaded metadata is a link metadata file, a data structure
    containing the materials and products is populated with the reported values.
-
-If more than one piece of link metadata are loaded due to the threshold value,
-their materials and products fields must exactly match.
-
-
-1. If the file is a layout file instead, the algorithm will recurse into that
-   layout, starting from step 5.
-1. If there is a layout file in conjunction with a link file (i.e., one of the
-   functionaries made a sublayout while others didn’t), then verification
-should fail. The reason as to why this happens is to avoid ambiguities between
-both versions of the supply chain.
+  1. If the file is a layout file instead, the algorithm will recurse into that
```

Comment:
To be consistent with the previous item in this list, we should say:
"If the loaded metadata file is a layout file instead, ..."

Reply:
Agreed. WIll do